demos/trino-subsea-data/create-table-in-trino.yaml

```
@@ -0,0 +1,83 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: create-subsea-multibeam-table-in-trino
+spec:
+  template:
+    spec:
+      containers:
+        - name: create-subsea-multibeam-table-in-trino
+          image: docker.stackable.tech/stackable/testing-tools:0.2.0-stackable24.7.0
+          command: ["bash", "-c", "python -u /tmp/script/script.py"]
+          volumeMounts:
+            - name: script
+              mountPath: /tmp/script
+            - name: trino-users
+              mountPath: /trino-users
+      volumes:
+        - name: script
+          configMap:
+            name: create-subsea-multibeam-table-in-trino-script
+        - name: trino-users
+          secret:
+            secretName: trino-users
+      restartPolicy: OnFailure
+  backoffLimit: 50
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: create-subsea-multibeam-table-in-trino-script
+data:
+  script.py: |
+    import sys
+    import trino
+
+    if not sys.warnoptions:
+        import warnings
+    warnings.simplefilter("ignore")
+
+    def get_connection():
+        connection = trino.dbapi.connect(
+            host="trino-coordinator",
+            port=8443,
+            user="admin",
+            http_scheme='https',
+            auth=trino.auth.BasicAuthentication("admin", open("/trino-users/admin").read()),
+        )
+        connection._http_session.verify = False
+        return connection
+
+    def run_query(connection, query):
+        print(f"[DEBUG] Executing query {query}")
+        cursor = connection.cursor()
+        cursor.execute(query)
+        return cursor.fetchall()
+
+    connection = get_connection()
+
+    run_query(connection, "CREATE SCHEMA IF NOT EXISTS hive.demo WITH (location = 's3a://demo/')")
+    run_query(connection, """
+        CREATE TABLE IF NOT EXISTS hive.demo.marispace (
+            footprint_x DOUBLE,
+            footprint_y DOUBLE,
+            water_depth DOUBLE,
+            data_point_density DOUBLE,
+            geometry VARBINARY
+        ) WITH (
+            external_location = 's3a://demo/',
```

I would suggest to have the same folders in S3 as in Trino for consistency - we also do this in other demos.
Remember to update `demos/trino-subsea-data/load-test-data.yaml` as well
```suggestion
            external_location = 's3a://demo/marispace',
```

---

https://github.com/stackabletech/demos/pull/88/commits/54d0d07248a6af38373ced722a97a1b2ca146a94